Sources/LocheckLogic/Validators/parseAndValidateXCStrings.swift

```diff
@@ -12,42 +12,32 @@ import Foundation
  Directly compare two `.strings` files
  */
 public func parseAndValidateXCStrings(
-    base: File,
-    translation: File,
+    base: [File],
+    translation: [File],
     translationLanguageName: String,
     problemReporter: ProblemReporter) {
-    problemReporter.logInfo("Validating \(translation.path) against \(base.path)")
-
-    let baseStrings = base.lo_lines.enumerated()
-        .compactMap {
-            LocalizedStringPair(
-                string: $0.1,
-                path: base.path,
-                line: $0.0 + 1)
+    let collectLines = { (files: [File], baseStringMap: [String: FormatString]) -> [LocalizedStringPair] in
```

Comment:
Looks like `baseStringMap` is unused?

Reply:
Hmm, this should have made the tests fail. I'll look into writing a regression test.